client/components/recycleAndReuseComponents/AddItemMultiSelect.js

```diff
@@ -1,12 +1,42 @@
-import { Button, Flex, Text } from "@chakra-ui/react";
-import React from "react";
+import { Button, Flex, Text, Alert, AlertIcon } from "@chakra-ui/react";
```

Comment:
revert all other changes - only main logic change should remain (as mentioned below)